### ISSUE TITLE
Missing Node Support

### ISSUE THREAD

**Issue by barbax** [sqids/sqids-javascript#19]
hashIDs supported Node.js use, which makes sense in my opinion for this library. Removing Blob from the js code may be the solution to be able to use it in Node as well.

**Comment by miquelfire** [comment]
What do you mean by missing Node support?

The package.json is written in a way that tells me it supports Node 100% (Both CJS and ESM, and CJS is only there for people who haven't switched to ESM yet in their code)

**Comment by barbax** [comment]
Line 584 of th elibrary:
` if (new Blob([alphabet]).size !== alphabet.length) {...`

 Blob is not a native nodejs type, so in our nestjs project I have this error ReferenceError: Blob is not defined
We are pushing a pull request to fix the issue in order not to use Blob

**Comment by barbax** [comment]
https://github.com/sqids/sqids-javascript/pull/20

**Comment by barbax** [comment]
After further digging we realized what were the root cause. Your manifest correctly states that the library need node 18+. We used it on a legacy node 16 project.

Node 18 (https://nodejs.org/en/blog/announcements/v18-release-announce) added support to Blob in the root scope, so Node 18+ apps will work with the current version of the library, legacy system wont.

Upgrading node fix the issue. Our pull request may be usefull for someone in need of made it work in legacy environment.

Thank You!
